### PR TITLE
some updates

### PR DESCRIPTION
**Pull request by m-1-k-3**
another little fixes everywhere pull request. It mainly tries to address the p05 issues and the S25-kernel problems

### CODE REVIEW

File: modules/P05_firmware_bin_extractor.sh [hunk 1]
```diff
@@ -60,8 +61,8 @@ P05_firmware_bin_extractor() {
     print_output "[*] Found $ORANGE$UNIQUE_BINS$NC unique executables and $ORANGE$BINS$NC executables at all."
   fi
 
-  # we can use $BINS as parameter -> it is usually gt 0
-  module_end_log "${FUNCNAME[0]}" "$BINS"
+  FILES_EXT=$(find "$FIRMWARE_PATH_CP" -xdev -type f | wc -l )
```
[the remark this answers is not in the document]
Comment (m-1-k-3) [reply modules/P05_firmware_bin_extractor.sh:64]: yes but in line 53 it is done before deb and ipk extraction. Now we are doing it with all extraction finished. It makes sense doing it only if FILES_EXT is zero.

File: emba.sh [hunk 2]
```diff
@@ -259,6 +260,7 @@ main()
         ;;
       l)
         export LOG_DIR="$OPTARG"
+        export TMP_DIR="$LOG_DIR""/tmp"
```
[the remark this answers is not in the document]
Comment (m-1-k-3) [reply emba.sh:263]: hmmm, I had some issues in updating hte TMP_DIR variable after setting a LOG_DIR. I will check this again

[the remark this answers is not in the document]
Comment (m-1-k-3) [reply emba.sh:263]: after removing it 

./modules/P05_firmware_bin_extractor.sh: Zeile 247: ./logs/tmp/binwalker.txt: Datei oder Verzeichnis nicht gefunden
and so on

[the remark this answers is not in the document]
Comment (p4cx) [reply emba.sh:264]: strange, than don't change it

File: emba.sh [hunk 3]
```diff
@@ -183,6 +183,7 @@ main()
   export YARA=1
 
   export MAX_EXT_SPACE=11000     # a useful value, could be adjusted if you deal with very big firmware images
+  export MAX_EXT_SPACE=21000     # a useful value, could be adjusted if you deal with very big firmware images
```
Comment (m-1-k-3) [inline emba.sh:186]: ups ... I will remove this

File: modules/P05_firmware_bin_extractor.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (p4cx) [reply modules/P05_firmware_bin_extractor.sh]: Ah, I understand - okey, then I will merge this PR 👍

**Review by p4cx — APPROVED** [review]

**Pull request merged**